removed dependency on junit's listener and used only surefire's repor…

alexgyori opened the pull request:
…ting to find test failures

august782 commented on nondex-maven-plugin/src/main/java/edu/illinois/nondex/plugin/CleanSurefireExecution.java:
```diff
@@ -79,63 +87,67 @@ public void run() throws MojoExecutionException {
+                for (ReportTestSuite report : parser.parseXMLReportFiles()) {
+                    for (ReportTestCase testCase : report.getTestCases()) {
+                        if (testCase.hasFailure()) {
+                            String testName = testCase.getFullName();
```
Is there a way to get the separate test class and test method names from the ReportTestCase?

august782 commented on nondex-maven-plugin/src/main/java/edu/illinois/nondex/plugin/CleanSurefireExecution.java:
```diff
@@ -79,63 +87,67 @@ public void run() throws MojoExecutionException {
-        }
-        return childNode;
+    private Xpp3Dom makeNode(String nodeName, String value) {
+        Xpp3Dom outputDir = new Xpp3Dom(nodeName);
```
Rename outputDir

alexgyori commented:
@august782 Please check my last commit; I added count incrementing in the extendZoneStrings, since it seems it was missing (on top of the refactorings that are just beautyfying the name and making them more readable for when stopping in the debugger). I also addressed the prior comments you had. LGTM?

alexgyori commented:
Fixes #11
